packages/utils/src/payment/abstract-payment-provider.ts

```diff
@@ -18,7 +18,7 @@ export abstract class AbstractPaymentProvider implements IPaymentProvider {
    * you can access it in the constructor. The options are passed as a second parameter.
    *
    * @param {MedusaContainer} container - An instance of `MedusaContainer` that allows you to access other resources, such as services, in your Medusa backend through [dependency injection](https://docs.medusajs.com/development/fundamentals/dependency-injection)
-   * @param {Record<string, unknown>} config - If this fulfillment provider is created in a plugin, the plugin's options are passed in this parameter.
+   * @param {Record<string, unknown>} options - If this payment processor is created in a plugin, the plugin's options are passed in this parameter.
```

Comment:
Same comment as adrien, would revert `options` back to `config`.

Reply:
@shahednasser I guess this became overly complicated, I just reverted it 😄